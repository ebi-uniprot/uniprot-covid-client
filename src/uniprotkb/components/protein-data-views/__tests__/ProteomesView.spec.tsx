import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import ProteomesEntryView from '../ProteomesView';
import ProteomesUIData from './__mocks__/proteomesUIData.json';

describe('ProteomesEntryView component', () => {
  test('should render', () => {
    const { asFragment } = render(
      <Router>
        <ProteomesEntryView data={ProteomesUIData} />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
