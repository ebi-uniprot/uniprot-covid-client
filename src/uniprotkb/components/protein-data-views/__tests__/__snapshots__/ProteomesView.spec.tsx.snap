// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProteomesEntryView component should render 1`] = `
<DocumentFragment>
  <div
    class="info-list"
  >
    <div
      class="info-list__item"
    >
      <div
        class="info-list__title"
      >
        <h5
          class="bold"
        >
          Identifier
        </h5>
      </div>
      <div
        class="info-list__content"
      >
        UP000005640
      </div>
    </div>
    <div
      class="info-list__item"
    >
      <div
        class="info-list__title"
      >
        <h5
          class="bold"
        >
          Component
        </h5>
      </div>
      <div
        class="info-list__content"
      >
        Chromosome 21
      </div>
    </div>
  </div>
</DocumentFragment>
`;
